test/fixtures/mson/inheritance-primitive.apib

```diff
@@ -0,0 +1,27 @@
+# GET /sample
++ Response 200 (application/json)
+    + Attributes (Message)
+
+# Data Structures
+
+# Message
++ created (Date3)
++ text (string)
+
+# Date (string)
+
+## Sample
+2012-12-00
+
+# Date2 (Date1)
+## Sample
+2012-12-01
```

Comment:
Probably, yes. But it is not important, so it can stay as is.
In final inheritance is used sample from `Date3` so test is able to check if it is inherited in required way